src/WebJobs.Extensions.DurableTask/LocalGrpcListener.cs

```diff
@@ -154,8 +155,22 @@ public override Task<Empty> Hello(Empty request, ServerCallContext context)
             {
                 try
                 {
+                    ActivitySource activityTraceSource = new ActivitySource("DurableTask.WebJobs");
```

Comment:
For context, the source name is "DurableTask.WebJobs" because we load the DurableTelemetryModule and it looks for sources that start with "DurableTask" ([related code link](https://github.com/Azure/durabletask/blob/93b2dde58366da95165fe8c8512b577c84628813/src/DurableTask.ApplicationInsights/DurableTelemetryModule.cs#L43)). I'm working on updating the listeners to add a new one ("WebJobs.Extensions.DurableTask") when we initialize the module so we can remove this "DurableTask.WebJobs" that's just used to test for now.